stubs/Pillow/PIL/TiffImagePlugin.pyi

```diff
@@ -71,34 +75,100 @@ class IFDRational(Rational):
     def limit_rational(self, max_denominator): ...
     def __hash__(self) -> int: ...
     def __eq__(self, other): ...
-    __add__: Any
-    __radd__: Any
-    __sub__: Any
-    __rsub__: Any
-    __mul__: Any
-    __rmul__: Any
-    __truediv__: Any
-    __rtruediv__: Any
-    __floordiv__: Any
-    __rfloordiv__: Any
-    __mod__: Any
-    __rmod__: Any
-    __pow__: Any
-    __rpow__: Any
-    __pos__: Any
-    __neg__: Any
-    __abs__: Any
-    __trunc__: Any
-    __lt__: Any
-    __gt__: Any
-    __le__: Any
-    __ge__: Any
-    __bool__: Any
-    __ceil__: Any
-    __floor__: Any
-    __round__: Any
+    # The following methods are aliased from fractions.Fraction:
```

Comment:
I think we could just do

```py
    __add__ = Fraction.__add__
    __radd__ = Fraction.__radd__
```

etc.

Reply:
I tried this at first, but got overload incompatible type errors with the overloaded functions. While we could probably just ignore those, I thought it would be safer to just copy the signature.

Reply:
Okay, let's keep it as you have it then